src/sinks/console.rs

```diff
@@ -203,27 +203,27 @@ mod test {
                 .into_iter()
                 .collect(),
             ),
-            kind: MetricKind::Incremental,
-            value: MetricValue::Counter { value: 100.0 },
-        });
+            MetricKind::Incremental,
+            MetricValue::Counter { value: 100.0 },
+        ));
         assert_eq!(
-            r#"{"name":"foos","namespace":"vector","timestamp":"2018-11-14T08:09:10.000000011Z","tags":{"Key3":"Value3","key1":"value1","key2":"value2"},"kind":"incremental","counter":{"value":100.0}}"#,
+            r#"{"name":"foos","namespace":"vector","tags":{"Key3":"Value3","key1":"value1","key2":"value2"},"timestamp":"2018-11-14T08:09:10.000000011Z","kind":"incremental","counter":{"value":100.0}}"#,
```

Comment:
Why is the reordering necessary here? Is it possibly flaky?

Reply:
This test is dependant on the exact ordering of the fields in the output JSON. That field order is in turn dependant on the order serde visits the struct fields, which has now changed. However, readers of JSON should not care what order the fields come in, so it's a harmless change in the output.